### PR TITLE
Fixes deprecation warning and updated CHANGELOG

### LINKED ISSUE #4339: Array to scalar deprecation warning
From nightly wheels cron CI:

2023-11-03T03:16:21.1300895Z   /home/runner/work/mdanalysis/mdanalysis/package/MDAnalysis/analysis/nuclinfo.py:302: DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation. (Deprecated NumPy 1.25.)
2023-11-03T03:16:21.1302468Z     phase_ang = (atan2(D, C) + (pi / 2.)) * 180. / pi

### PR DESCRIPTION
**Pull request by tanishy7777**
Fixes #4339 

Changes made in this Pull Request:
 - Changed the function atan2() to np.atan2() to avoid converting array to scalar

PR Checklist
------------
 - [x] Tests?
 - [ ] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

## Developers certificate of origin
- [x] I certify that this contribution is covered by the LGPLv2.1+ license as defined in our [LICENSE](https://github.com/MDAnalysis/mdanalysis/blob/develop/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).

----
📚 Documentation preview 📚: https://mdanalysis--4831.org.readthedocs.build/en/4831/

### CODE REVIEW

File: package/CHANGELOG [hunk 1]
```diff
@@ -14,11 +14,12 @@ The rules for this file:
 
 
 -------------------------------------------------------------------------------
-??/??/?? IAlibay, ChiahsinChu, RMeli
+??/??/?? IAlibay, ChiahsinChu, RMeli, tanishy7777
 
  * 2.9.0
 
 Fixes
+ * Fixes deprecation warning Array to scalar convertion. Replaced atan2() with np.arctan2()
```
Comment (orbeckst) [inline package/CHANGELOG]: please add issue as a reference (see other entries as examples)

File: package/CHANGELOG [hunk 2]
```diff
@@ -14,11 +14,12 @@ The rules for this file:
 
 
 -------------------------------------------------------------------------------
-??/??/?? IAlibay, ChiahsinChu, RMeli
+??/??/?? IAlibay, ChiahsinChu, RMeli, tanishy7777
```
Comment (orbeckst) [inline package/CHANGELOG:17]: Also add yourself to AUTHORS as you're a first time contributor.

File: package/MDAnalysis/analysis/nuclinfo.py [hunk 3]
```diff
@@ -299,7 +299,7 @@ def phase_cp(universe, seg, i):
          + (r3_d * cos(4 * pi * 2.0 / 5.0)) + (r4_d * cos(4 * pi * 3.0 / 5.0))
          + (r5_d * cos(4 * pi * 4.0 / 5.0))) * sqrt(2.0 / 5.0)
 
-    phase_ang = (atan2(D, C) + (pi / 2.)) * 180. / pi
```
Comment (orbeckst) [inline package/MDAnalysis/analysis/nuclinfo.py:302]: Remove the imported `atan2` from the import line at the top
```python
from math import pi, sin, cos, atan2, sqrt, pow
```

**Review by orbeckst — CHANGES_REQUESTED** [review]
Thank you for the fix. In addition to the comments inline, could you please also demonstrate that your changes fix the issue? Can you please show that before your changes, the warning is issued and that afterwards it is gone, e.g., by running `pytest MDAnalysisTests/analysis/test_nuclinfo.py` and showing _all_ the output?

**Comment by tanishy7777** [comment]
> Thank you for the fix. In addition to the comments inline, could you please also demonstrate that your changes fix the issue? Can you please show that before your changes, the warning is issued and that afterwards it is gone, e.g., by running `pytest MDAnalysisTests/analysis/test_nuclinfo.py` and showing _all_ the output?

Before
image

After
image

File: package/MDAnalysis/analysis/nuclinfo.py (hunk 3 above)
Comment (tanishy7777) [reply package/MDAnalysis/analysis/nuclinfo.py:302]: > Remove the imported `atan2` from the import line at the top
> 
> ```python
> from math import pi, sin, cos, atan2, sqrt, pow
> ```


Actually on line 371 atan2 is being used. But that is not generating any warnings so I didn't change that.
```python
phase_ang = atan2(B, A) * 180. / pi
```

Comment (orbeckst) [reply package/MDAnalysis/analysis/nuclinfo.py:302]: Can you replace that one, too, please, for consistency?

Comment (tanishy7777) [reply package/MDAnalysis/analysis/nuclinfo.py:302]: > Can you replace that one, too, please, for consistency?

Yes no problem. I have made the necessary changes

**Review by orbeckst — APPROVED** [review]
Looks good, thanks for the fix!

**Pull request merged**